src/common/avatars/BaseAvatar.js

```diff
@@ -40,7 +40,7 @@ class BaseAvatar extends React.PureComponent<Props> {
         style={[containerStyle, styles.avatar]}
       >
         <CachedImage
-          innerRef={imageRef}
+          ref={imageRef}
```

Comment:
This is a really weird pattern. Usually `ref` is to get the ref. Not to pass it in

Reply:
idk, I saw this ton of times in external libs. What is the better way to get some static props of one of the component children?